src/System.Private.CoreLib/shared/System/Resources/ResourceReader.cs

```diff
@@ -96,7 +96,20 @@ public sealed class ResourceReader : IResourceReader
         private unsafe int* _namePositionsPtr;  // If we're using UnmanagedMemoryStream
         private Type[] _typeTable;    // Lazy array of Types for resource values.
         private int[] _typeNamePositions;  // To delay initialize type table
-        private int _numResources;    // Num of resources files, in case arrays aren't allocated.        
+        private int _numResources;    // Num of resources files, in case arrays aren't allocated.
+
+        private bool _permitDeserialization;  // bool indicating if this ResourceReader deserialize BinaryFormatted resources
```

Comment:
Nit: "bool indicating if this ResourceReader deserialize" => "bool indicating if this ResourceReader can deserialize"

Reply:
nit: readonly